Sources/BraintreeShopperInsights/BTShopperInsightsClientV2.swift

```diff
@@ -56,6 +56,19 @@ public class BTShopperInsightsClientV2 {
         return try await updateCustomerSessionAPI.execute(request, sessionID: sessionID)
     }
     
+    /// Generates customer recommendations
+    /// - Parameters:
+    ///    - request: request type `BTCustomerSessionRequest`
+    ///    - sessionID: The shopper session ID
+    /// - Warning: This method is currently in beta and may change or be removed in future releases.
+    public func generateCustomerRecommendations(
+        request: BTCustomerSessionRequest,
+        sessionID: String
+    ) async throws -> BTCustomerRecommendationsResult {
+        let api = BTCustomerRecommendationsAPI(apiClient: apiClient)
```

Comment:
To match the rest of this module:
```suggestion
        let customerRecommendationsAPI = BTCustomerRecommendationsAPI(apiClient: apiClient)
```